src/download-helpers.js

```diff
@@ -132,12 +132,19 @@ export async function downloadReport(url, format, width, height, filename, authT
       }
     }
 
-    await browser.close();
-
     const timeCreated = time.valueOf();
     const data = { timeCreated, dataUrl: buffer.toString('base64'), };
-
     await readStreamToFile(data.dataUrl, filename, format);
+
+    if (transport !== undefined) {
+      let emailTemplateImageBuffer = await page.screenshot({
+        fullPage: true,
+      });
+      const data = { timeCreated, dataUrl: emailTemplateImageBuffer.toString('base64'), };
+      await readStreamToFile(data.dataUrl, 'email_body.png', FORMAT.PNG);
```

Comment:
email doesn't support csv?

Reply:
CSV is attached with email and this image is added in the html body. Check this video and let me know if we should change something

https://user-images.githubusercontent.com/56703525/214384721-6bc77e02-9b01-47ce-b3fc-6bd53486ecfa.mov

Reply:
got it, looks good